libcontainer/network_linux.go

```diff
@@ -98,3 +101,149 @@ func (l *loopback) attach(n *configs.Network) (err error) {
 func (l *loopback) detach(n *configs.Network) (err error) {
 	return nil
 }
+
+// getNetns return the path to network namespace
+// or an empty string if it is not available.
+func getNetns(config *configs.Config) string {
```

Comment:
In fact, this function is not necessary because we can use `config.Namespaces.PathOf(...)` directly instead.

Reply:
does it return empty if there is no network namespace? I added the helper because we need to discriminate between containers with and without network namespace

Reply:
done